src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -97,6 +97,27 @@ public void IsPinOpenTest()
             }
         }
 
+        [Fact]
+        public void IsPinOpenTest2()
```

Comment:
I'd rename this and previous test so that it's more obvious what's the difference between this and previous test